guides/common/modules/proc_configuring-the-cleaning-unused-tasks-feature.adoc

```diff
@@ -10,26 +10,25 @@ By default, {Project} executes a cron job that cleans tasks every day at 19:45.
 * Tasks that have run successfully and are older than thirty days
 * All tasks that are older than a year
 
-.For {Project}s Upgraded from Previous Versions
-ifndef::orcharhino[]
-Until https://bugzilla.redhat.com/show_bug.cgi?id=1788615[BZ#1788615] is resolved, this functionality works only on fresh installations of {ProjectXY} and later.
-endif::[]
-If you upgrade {Project} from previous versions, this functionality is disabled by default.
-To enable {Project} to perform regular cleaning, enter the following command:
+You can configure the cleaning unused tasks feature using these options:
 
+* To configure the time at which {Project} runs the cron job, set the `--foreman-plugin-tasks-cron-line` parameter to the time you want in cron format.
+For example, to schedule the cron job to run every day at 15:00, enter the following command:
++
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --foreman-plugin-tasks-automatic-cleanup true
+# {foreman-installer} --foreman-plugin-tasks-cron-line "00 15 * * *"
 ----
-
-Optionally use this procedure to adjust the configuration to serve your needs.
-
-.Procedure
-. Optional: To configure the time at which {Project} runs the cron job, set the `--foreman-plugin-tasks-cron-line` parameter to the time you want in cron format.
-For example, to schedule the cron job to run every day at 15:00, enter the following command:
+* To configure the period after which {Project} deletes the tasks, edit the `:rules:` section in the `/etc/foreman/plugins/foreman-tasks.yaml` file.
```

Comment:
Should there be an example in code block for this?

Reply:
Hey @adamlazik1 I just copied the original text, but good idea! Let me check this on my instance and add an example.

Reply:
I thought the same, but then also thought it'd spread the scope of this PR too wide?
(Not saying it shouldn't be done, just not here and today)

Reply:
I am fine with either way, rest of the PR looks good to me. @AkshayGadhaveRH your choice.

Reply:
Hey @adamlazik1 @evgeni so I checked the option and tried to insert an example in the code, but it might need some tweaking so that it renders correctly. Let's add this at a later time?